### DISCUSSION TITLE
Both original and AW file on Octoprint

### DISCUSSION THREAD

**Discussion by phodara** [FormerLurker/ArcWelderPlugin#244] [Q&A]
Neither a problem or a real issue, just a question.  I am using Arc-Welder's Octoprint Plugin, my slicer is Cura, everything is up to date.  When I slice my model and push it out to Octoprint, Octoprint automatically loads the original GCode file not the AW file.  Is there a way to get Octoprint to by default load the AW file?

**Comment by Sheridan-Tech** [comment]
Apparently the way to do this is to tell cura not to print automatically and just upload. Then if you are running the development version of arc welder and have "print after processing" it will automatically load that file and print it.
